index.html

```diff
@@ -66,7 +66,7 @@ <h3>LET’S CONNECT</h3>
     <!--Work section -->
     <section id="work">
 
-      <div class="work">
+      <div class="work" class="modalContainer">
```

Comment:
- [ ] It is a project requirement that you refactor your project section here labelled as *<section id="work">` to be generated dynamically. Kindly implement this 😄

Reply:
- [ ] As requested by the previous reviewer the `Projects` section of your project should be created dynamically using JavaScript. Kindly create an array of objects to hold the information for each card, then loop through the array to display the content dynamically

e.g
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```